docs/content/setup/linux.md

```diff
@@ -9,6 +9,15 @@ tags: ["setup", "AppImage", "linux"]
 
 ## Supported operating systems
 
+The Infection Monkey AppImage package is supported on the following operating systems (not an
+exhaustive list):
+- Debian  <!-- TODO: which? -->
+- Kali  <!--- TODO: which? -->
+- Ubuntu 18.04
+
+The following do NOT support the Infection Monkey AppImage package (not an exhaustive list):
+- Ubuntu 16.04
+
```

Comment:
Let's get rid of this. 16.04 has been EOL since April. Unless there's a common OS that should work that we know for sure doesn't, I don't think we need this section.